xml/src/main/java/org/ehcache/xml/XmlConfiguration.java

```diff
@@ -319,6 +321,10 @@ private void parseConfiguration()
                   .queueSize(writeBehind.maxQueueSize()));
         }
       }
+      if (cacheDefinition.resilienceStrategy() != null) {
+        final Class<ResilienceStrategy<?, ?>> resilienceStrategyClass = (Class<ResilienceStrategy<?, ?>>) getClassForName(cacheDefinition.resilienceStrategy(), cacheClassLoader);
```

Comment:
If you remove the useless `final`s, I would be happy

Reply:
Okay.